debian/mumbledj.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+After=mumble-server.service
```

Comment:
Does this line assume that the Mumble server is running on the same machine as MumbleDJ?